microraiden/microraiden/config.py

```diff
@@ -3,8 +3,8 @@
 from eth_utils import denoms
 
 API_PATH = "/api/1"
-GAS_LIMIT = 250000
-GAS_PRICE = 20 * denoms.gwei
+GAS_LIMIT = 130000
+GAS_PRICE = 50 * denoms.gwei
```

Comment:
What was the reason for increasing the gas price? Running the Ropsten tests faster?
I see `GAS_PRICE` is used for `create_signed_contract_transaction` when closing all channels. This should be something configurable by the user through a cli option in this case.
Do we use the gas price for anything else production-related?

Reply:
Yes, faster inclusion on Ropsten. Should probably decrease that again or use different prices for testnets and mainnet again.

I'd separate a new CLI option into a different PR though (+issue).

Reply:
we should also discuss having gas prices configurable per-network, as is the case with block number

Reply:
agreed, a separate PR would be better. I created an issue https://github.com/raiden-network/microraiden/issues/354